components/Typography.jsx

```diff
@@ -64,3 +64,33 @@ export const Code = styled("code", {
   borderWidth: 1,
   borderRadius: "$3",
 });
+
+export const IconButton = styled("button", {
+  marginLeft: 5,
+  marginRight: 5,
+
+  paddingInline: "$3",
+  paddingBlock: "$3",
+
+  color: "$white",
+  backgroundColor: "$orange",
+
+  borderColor: "$orange",
+  borderStyle: "solid",
+  borderWidth: 1,
+  borderRadius: "$3",
+
+  "& > svg": {
+    width: "20px",
+    height: "20px",
+  },
```

Comment:
This component shouldn't be opinionated about this. Also the radix icons are designed to be 15x15. Does it look weird if they are smaller?